backend/app/routes/broadcast/getBroadcasts.js

```diff
@@ -0,0 +1,43 @@
+const to = require('await-to-js').default;
+const Broadcast = require('../../models/Broadcast');
+const { ErrorHandler } = require('../../../helpers/error');
+const constants = require('../../../constants');
+
+module.exports = async (req, res, next) => {
+  const page = parseInt(req.query.page, 10);
+  const limit = parseInt(req.query.limit, 10);
+  const startIndex = (page - 1) * limit;
+  const endIndex = page * limit;
+  const results = {};
+  if (!page || !limit) {
```

Comment:
you must assume default value for page to be 1

Reply:
ok